testfiles/5ZoneWLHPPlantLoopTower.idf

```diff
@@ -1819,7 +1819,7 @@
     2964,                    !- Lighting Level {W}
     ,                        !- Watts per Zone Floor Area {W/m2}
     ,                        !- Watts per Person {W/person}
-    0.2,                     !- Return Air Fraction
+    ,                        !- Return Air Fraction
```

Comment:
Because these are zonehvac WLHPs which don't have any return air flow, so it was generating a warning about that and putting the heat gain into the zone anyway.